services/core-api/app/api/parties/party_appt/resources/mine_party_appt_resource.py

```diff
@@ -149,7 +149,7 @@ def post(self, mine_party_appt_guid=None):
             if tsf is None:
                 raise NotFound('TSF not found')
 
-            if party.party_type_code is not "PER":
+            if party.party_type_code != "PER":
                 raise Forbidden(f"A contact for {mine_party_appt_type_code} must be a person not an organization")
```

Comment:
just while you're in here... should these (I think there's about 3 here) be `Forbidden` or should they be `BadRequest`? I was confused to see the 403s coming back!